### DISCUSSION TITLE
How to "Heal Network"

### DISCUSSION THREAD

**Discussion by Juggler00** [zwave-js/zwave-js-ui#4020] [Q&A] [answered]
I'm running the latest version of Z-Wave JS UI. There used to be a way to "heal" the network. I could be missing it, but I no longer see this as an option. Can someone point me in the right direction?

zwave-js-ui: 9.26.0.47f4c89
zwave-js: 13.10.3

**Comment by kpine** [comment, marked as answer]
It's called "Rebuild Routes" and "Rebuild Node Routes" in the Advanced Actions menu.

**Reply by Juggler00** [reply]
Thank you

**Reply by ghost** [reply]
Not there any more

**Reply by robertsLando** [reply]
@djashjones I moved it by error in the advanced menu when you have selected nodes, wait for next patch release please 🙏🏼

**Reply by ghost** [reply]
lol, ok,, no worries.
